### DISCUSSION TITLE
Images for the prototype presentation

### DISCUSSION THREAD

**Discussion by davide-f** [pypsa-meets-earth/pypsa-earth#229] [Q&A] [answered]
Hello,

On January 27th, we are presenting our initiative to the world, and in this discussion we are asking for your suggestions and comments for achieving the best result and maximize the the output of the event.

**Good images** are determinant success factors.
Any help in terms of **recommendation, shared scripts for plotting or nice plots** aligned with the purpose is welcomen.

Below, we describe the type of plots we aim to produce and we'll keep the plots updated with our current version for receiving your thoughts.

Let's PyPSA meets Africa&Earth grow larger together!

Desired images:
- [x] OSM extraction tool: show the downloaded network data
- [x] Powerplants: show the available powerplants ideally by type and GIS location
- [ ] Weather data and resource potential [with/without protected areas]
- [x] Demand data over years
- [x] Plots of clustered and simplified network
- [x] Augmented network [connect mini-grids and enhance studies on network]
- [x] Optimized PyPSA Africa model
- [x] Extensions for sector coupled and pathway optimization
- [x] Earth model

**Comment by davide-f** [comment, marked as answer]
@Hazem-IEG @euronion @fneum , any help in terms of similar plots/scripts or anything you feel may help would be much appreciated.
Me and @pz-max  will keep this post updated with the current advancements

**Reply by pz-max** [reply]
Great list @davide-f , :100: 
I will take care of the **augmented network** and the **optimized Africa model** in the next two days. Adding these points ideally also to the plotting_summary if there are not already there :)

**Comment by davide-f** [comment]
Regarding the OSM data and the earth model, I've already taken care of them. Added in the dots

**Comment by pz-max** [comment]
- 420 nodes, pre-clustered network (we could create 3 voltage level classes to make it look a bit better/data richer)
<img src="https://user-images.githubusercontent.com/61968949/151141990-ded23964-4b02-4534-8c63-26649c52ae32.png" width="600" height="600">

- 120 cluster_pre_augmented_network
<img src="https://user-images.githubusercontent.com/61968949/151145882-24f095f2-c498-4a4d-b124-676c9864b1a0.png" width="800" height="800">

- 120 cluster_post_augmented_network (in white are the links)
<img src="https://user-images.githubusercontent.com/61968949/151146046-b339046f-34a6-43dd-9791-82bc56880806.png" width="800" height="800">

- 120 node optimized network (network is from Davide and is representing the clustered **not** augmented network). Shows only expanded lines in a 100% renewable scenario where only wind and solar is optimized. The network was here not meshed with the augmentation technique. Therefore some isolated buses exist. Further the amount of buses which were assigned to each country was depending on the demand. That's why demand rich countries have extreme high resolution. We implemented a new bus distribution technique wich can use population or GDP as factors instead of the demand.

<img src="https://user-images.githubusercontent.com/61968949/151148040-56701316-eea6-4833-9071-0f9c9c4458c2.png" width="700" >

**Reply by jarry7** [reply]
Looks cool @pz-max - I assume that the underlying shapes in the graphic are the administrative layers within each country (adm1/adm2).  Perhaps as an introductory map/graphic, the underlying shapefile could just include country-by-country (just for ease of presentation to an audience)?  Following this, the underlying shapes could be presented?

**Reply by pz-max** [reply]
Good point. We will add an image with country shapes :)

**Comment by davide-f** [comment]
OSM images
![image](https://user-images.githubusercontent.com/67809479/151162187-194b83de-4824-4001-9b1a-4bfcdb954132.png)
![image](https://user-images.githubusercontent.com/67809479/151162220-f028c8c6-3a6e-4bd8-87e3-4cce2d08c61f.png)

**Reply by pz-max** [reply]
@davide-f  cleaned or raw version?

**Reply by davide-f** [reply]
> @davide-f cleaned or raw version?

They are raw

**Reply by hazemakhalek** [reply]
Great work @davide-f and @pz-max! 
I am planning to work on the plotting functions soon and since you're preparing the slides now I thought I might try to make some of the plots clearer. I can't find the notebook used though, especially the one used for the osm images. Are they in the repo?

**Comment by pz-max** [comment]
Regarding the world picture @davide-f  :) . It seems that some of the gulf states are not plotted. No need to do it if there is no time left
![middle-east-power-data-raw](https://user-images.githubusercontent.com/61968949/151199386-dcc1170d-ce6e-4926-a333-ab7088281c69.png)
